tests/vppoutputencode.h

```diff
@@ -81,7 +88,20 @@ class VppOutputEncode : public VppOutput
 
 private:
     void initOuputBuffer();
+    void getFrameDelay(VideoEncOutputBuffer m_outputBuffer);
+    void getBitRate(VideoEncOutputBuffer m_outputBuffer);
     const char* m_mime;
+    int64_t m_MaxLatency;
```

Comment:
This expose too much details to VppOutputEncode , could we use other class like EncoderLantency to hide all details in class? you can refer to
```
class FpsCalc
{
public:
    FpsCalc();
    void addFrame();
    void log();
private:
    int m_frames;
    uint64_t m_start;
    uint64_t m_netStart;
    static const int NET_FPS_START = 5;
};
```
all private member are details, caller do not need know this